### PR TITLE
Add example for storage container

### PR DESCRIPTION
**Pull request by neil-yechenwei**
fixes https://github.com/terraform-providers/terraform-provider-azurerm/issues/8214

### CODE REVIEW

File: examples/storage/storage-container/main.tf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+resource "azurerm_storage_account" "example2" {
+  name                = "${var.prefix}storageacct2"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  account_tier              = "Standard"
+  account_kind              = "StorageV2"
+  account_replication_type  = "LRS"
+  enable_https_traffic_only = true
+  access_tier               = "Cool"
```
Comment (tombuildsstuff) [inline examples/storage/storage-container/main.tf:38]: this should probably be

```suggestion
  access_tier               = "Hot"
```

**Review by tombuildsstuff — APPROVED** [review]
one comment to fix but this otherwise LGTM 👍

File: examples/storage/storage-container/main.tf (hunk 1 above)
Comment (neil-yechenwei) [reply examples/storage/storage-container/main.tf:38]: updated

**Comment by neil-yechenwei** [comment]
@tombuildsstuff , thanks for your comment. I've updated code.

**Pull request merged**